test/e2e/antctl_test.go

```diff
@@ -162,7 +139,7 @@ func testAntctlAgentLocalAccess(t *testing.T, data *TestData) {
 
 func runAntctlPod(t *testing.T, data *TestData, podName string, antctlServiceAccountName string, antctlImage string, covDir string) {
 	b := NewPodBuilder(podName, data.testNamespace, antctlImage).WithServiceAccountName(antctlServiceAccountName).
-		WithContainerName("antctl").WithCommand([]string{"sleep", "3600"}).
+		WithContainerName("antctl").WithEnv([]corev1.EnvVar{{Name: "GOCOVERDIR", Value: covDir}}).WithCommand([]string{"sleep", "3600"}).
```

Comment:
`covDir` is also `/tmp/coverage` now which I suppose it is a default value of `GOCOVERDIR`? Do we need `WithEnv([]corev1.EnvVar{{Name: "GOCOVERDIR", Value: covDir}})`?

Reply:
Not sure I fully understand the question, but we set the `GOCOVERDIR` env variable as part of the Dockerfile now. It has to be set to `/tmp/coverage` anyway because it has to point to an existing directory.